pkg/docker/config/config.go

```diff
@@ -739,13 +739,24 @@ func stripScheme(url string) string {
 	return stripped
 }
 
+// normalizeAuthFileKey takes a key, converts it to a host name and normalizes
+// the resulting registry.
+func normalizeAuthFileKey(key string, legacyFormat bool) string {
+	stripped := stripScheme(key)
+	if legacyFormat || stripped != key {
+		stripped = convertToHostname(key)
```

Comment:
This is the only caller of `convertToHostname`, and we have already stripped the scheme, only for `convertToHostname` to try doing it again. If `convertToHostname` were inlined, we could remove that redundancy.

(And then, absolutely optionally, inline `stripScheme` into the only caller.)

Reply:
Yep makes sense, fixed as suggested :+1: